spec/features/allocation_information_spec.rb

```diff
@@ -146,4 +195,34 @@ def create_allocation(offender_no)
       recommended_pom_type: recommended_pom_type
     )
   end
+
+private
+
+  def stub_api_calls_for_prison_allocation_path(sentence_start_date:, conditional_release_date:, automatic_release_date:, hdced:, release_date:)
+    stub_request(:post, "https://gateway.t3.nomis-api.hmpps.dsd.io/auth/oauth/token?grant_type=client_credentials").
+            to_return(status: 200, body: {}.to_json, headers: {})
+
+    stub_request(:get, "https://gateway.t3.nomis-api.hmpps.dsd.io/elite2api/api/prisoners/#{offender_no}").
+      to_return(status: 200, body: [{ offenderNo: offender_no, title: "MR", firstName: "OZULLIRN", middleNames: "TESSE", lastName: "ABBELLA", dateOfBirth: "1980-08-15", gender: "Male", sexCode: "M", nationalities: "sxiVsxi", currentlyInPrison: "Y", latestBookingId: "1153753", latestLocationId: "LEI", latestLocation: "Leeds(HMP)", internalLocation: "LEI-B-4-021", pncNumber: "97/39395W", croNumber: "043735/97V", ethnicity: "White:Eng./Welsh/Scot./N.Irish/British", birthCountry: "England", religion: "EfJSmIEfJSm", convictedStatus: "Convicted", imprisonmentStatus: "SENT03", receptionDate: "2016-11-26", maritalStatus: "sjedbztIaJRRzRZVIYadsjedbztIaJRRzRZVIYa" }].to_json)
```

Comment:
This is a large enough data structure that I think it's worth sticking it somewhere else - either as a variable somewhere in this file, or even in a flat file fixture.

That said, this problem is repeated across the app, so may be better to try and address once we have more context from working on it for longer.

Reply:
Yeah, I agree- depending on our priorities right now, I'm happy to do either.